cocotb/__init__.py

```diff
@@ -113,6 +104,22 @@ def _initialise_testbench(root_name):
 
     process_plusargs()
 
+    # Seed the Python random number generator to make this repeatable
+    seed = os.getenv('RANDOM_SEED')
+
+    if seed is None:
+        if 'ntb_random_seed' in plusargs:
+            seed = eval(plusargs['ntb_random_seed'])
```

Comment:
Is there a good reason to use `eval` here rather than just passing an int?

`vsim +ntb_random_seed="import shutil; shutil.rmtree(\"/\")"`